src/wallet/rpcdump.cpp

```diff
@@ -1154,12 +1154,12 @@ static UniValue ProcessImportDescriptor(ImportData& import_data, std::map<CKeyID
 
     const UniValue& priv_keys = data.exists("keys") ? data["keys"].get_array() : UniValue();
 
-    // Expand all descriptors to get public keys and scripts.
-    // TODO: get private keys from descriptors too
+    // Expand all descriptors to get public keys and scripts, and private keys if available.
```

Comment:
nit: Maybe take opportunity to add `importmulti` as a supporting RPC in `descriptors.md`.
You may also add a descriptor import example in `importmulti`